Makefile

```diff
@@ -46,6 +46,7 @@ LLVM_AS = $(LLVM_BINDIR)/llvm-as
 LLVM_NM = $(LLVM_BINDIR)/llvm-nm
 LLVM_CXX_FLAGS = -std=c++11  $(filter-out -O% -g -fomit-frame-pointer -pedantic -W% -W, $(shell $(LLVM_CONFIG) --cxxflags | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):/\/\1/g;s/-D/ -D/g;s/-O/ -O/g'))
 OPTIMIZE ?= -O3
+OPTIMIZE_FOR_BUILD_TIME ?= -O0
```

Comment:
Do we want to make a similar fix for CMake?

Reply:
As far as I can tell, we aren't setting the optimization level for tests at all in the CMake build, except for external_code_extern.cpp...

Reply:
...which implies the performance tests are dubious for CMake. Opened https://github.com/halide/Halide/issues/2566